frontend/src/modules/Dashboard/Components/Dashboard.tsx

```diff
@@ -41,15 +60,54 @@ export const Dashboard = () => {
       })
   }, [userEmail])
 
+  const MenuItemtheme = createMuiTheme({
+    typography: {
+      fontSize: 16,
+      fontFamily: 'Montserrat',
+    },
+  })
+
+  const Menutheme = createMuiTheme({})
+
   return (
     <StyledOuterContainer>
       <StyledContainer>
         <StyledHeaderMenu>
           <StyledLogo />
-          <StyledName>
-            {useAppSelector((state) => state.auth.user?.displayName)}
-            <StyledArrowDown />
-          </StyledName>
+          <Button
+            id="logout-button"
+            aria-controls="logout-menu"
+            aria-haspopup="true"
+            aria-expanded={open ? 'true' : undefined}
```

Comment:
I'm guessing this was adapted from the Menu examples on the MUI documentation. [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) are for accessibility, which we should keep in mind while developing. However, ARIA needs to be used very intentionally as it can also make the experience worse if used wrong (I don't know enough at the moment to comment if this is a right/wrong use of ARIA).